crates/cli_utils/src/helpers.rs

```diff
@@ -82,7 +82,7 @@ where
     I: IntoIterator<Item = S>,
     S: AsRef<OsStr>,
 {
-    run_with_stdin(&path_to_roc_binary(), args, &[])
+    run_roc_with_stdin(&path_to_roc_binary(), args, &[])
```

Comment:
`run_with_stdin` was written so that any runnable file could be passed in. Currently it is only used for running roc, so I wanted the name to reflect that. This allows us to understand whats is going in more quickly if this function is shown in a stacktrace.